media/css/custom.scss

```diff
@@ -0,0 +1,494 @@
+}
+
+.proper-content {
+  padding-top: $footer-height;
+  // 60px to make the container go all the way to the bottom of the topbar
+}
+
+// Set the fixed height of the footer here
+
+#push,
+footer {
+  height: $footer-height;
+}
+
+footer {
+  background-color: $lighter-gray;
+}
+
+// Lastly, apply responsive CSS fixes as necessary
+@media (max-width: 767px) {
+  footer {
+    margin-left:  -20px;
+    margin-right: -20px;
+    padding-left:  20px;
+    padding-right: 20px;
+  }
+}
+
+.container {
+  .credit {
+    margin: 20px 0;
+  }
+}
+
+#feed-range {
+  cursor: pointer;
+  padding: 5px 10px;
+  border: 1px solid $white;
+  &:hover {
+    background-color: $widget-link-highlight;
+  }
+}
+
+#rssdata ul.rss-items {
+  display: none;
+  margin:  0;
+  padding: 0;
+  li {
+    display: block;
+    margin:  0;
+    padding: 0;
+  }
+  a {
+    margin: 0;
+    display: block;
+    font: 15px / 16px $font-stack;
+    padding: 5px;
+    background: $sidebar-background;
+    color: $white;
+    text-decoration: none;
+    border-bottom: 1px solid $sidebar-border-color;
+    &:hover {
+      background: $sidebar-link-highlight;
+      color: $white;
+      text-decoration: none;
+    }
+  }
+}
+
+#rsstitle {
+  padding: 6px;
+  color: $white;
+  font: 17px / 18px bold $font-stack;
+  text-decoration: none;
+  border-bottom: 2px solid $sidebar-border-color;
+  height: 25px;
+}
+
+#rssbottom {
+  padding: 2px 6px;
+  background: $sidebar-background;
+  color: $white;
+  text-decoration: none;
+  border-top: 2px solid $sidebar-border-color;
+  height: 20px;
+}
+
+#twitter ul.twits {
+  display: none;
+  margin:  0;
+  padding: 0;
+  li {
+    display: block;
+    margin:  0;
+    padding: 0;
+  }
+  a.tw {
+    margin: 0;
+    display: block;
+    font: 15px / 16px $font-stack;
+    padding: 5px;
+    background: $sidebar-background;
+    color: $white;
+    text-decoration: none;
+    border-bottom: 1px solid $sidebar-border-color;
+    &:hover {
+      background: $sidebar-link-highlight;
+      color: $sidebar-link-highlight;
+      text-decoration: none;
+    }
+  }
+}
+
+#twitter_title {
+  padding: 6px;
+  color: $white;
+  font: 17px / 18px bold $font-stack;
+  text-decoration: none;
+  border-bottom: 2px solid $sidebar-border-color;
+}
+
+#twitter_bottom {
+  padding: 2px 6px;
+  background: $sidebar-background;
+  color: $white;
+  text-decoration: none;
+}
+
+.white {
+  color: $white;
+}
+
+a.white {
+  color: $white;
+  &:hover {
+    color: $white;
+  }
+}
+
+.article {
+  .thumbnail {
+    margin: 15px;
+  }
+}
+
+a {
+  color: $sidebar-background;
+  &:hover {
+    color: $sidebar-link-highlight;
+  }
+  &:focus {
+    outline: thin dotted $sidebar-link-highlight;
+    outline-offset: -2px;
+  }
+}
+
+.well {
+  background-color: $sidebar-background;
+  li a {
+    color: $white;
+  }
+}
+
+a.thumbnail {
+  background-color: $white;
+  &:hover {
+    background-color: $sidebar-link-highlight;
+  }
+}
+
+#sidebar {
+  .well, h2, a  {
```

Comment:
Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space